0-bootstrap/sa.tf

```diff
@@ -82,8 +82,7 @@ locals {
       "roles/dns.admin",
     ],
     "proj" = [
-      "roles/resourcemanager.folderViewer",
-      "roles/resourcemanager.folderIamAdmin",
+      "roles/resourcemanager.folderAdmin",
```

Comment:
This permission is needed because now 4-projects step creates the proper subfolders by example BUs (before all the folders were created at 2-environments).
`folderAdmin` because we need permissions to create and destroy.